### DISCUSSION TITLE
Do you have any experience using gRPC for internal microservices communication?

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#204] [Q&A] [answered]
I am considering using gRPC for the communication among microservices in the Kubernetes cluster.
Do you have any previous experience / compatibility issues / warnings to suggest?

Thanks in advance!

**Comment by frengooo** [comment, marked as answer]
Hi @Lucascanna, I tried it on my private project and it worked.
I just created two Node.JS example microservices (a client and a server) and followed the official documentation:
https://docs.mia-platform.eu/docs/development_suite/use-cases/grpc
